src/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ReactNode } from "react";
+import { render, fireEvent, act, screen } from "@testing-library/react";
+import { ReactSearch } from ".";
+import * as SearchInput from "./SearchInput";
+import * as SearchResult from "./SearchResult";
```

Comment:
Nit: We can make our interfaces more predictable by consistently preferring named exports over default exports. E.g. replacing this with:

```js
import { SearchInput } from "./SearchInput";
import { SearchResult } from "./SearchResult";
```

Reply:
Oh this is only for the test, in order to be able to use jest's `spyOn` syntax:

```
// Where object is the star import, and method is the named export (i.e. SearchInput)
jest.spyOn(<object>, <method>);
```

If you know of a better way to do this, would definitely love to know more!